src/components/Home/HomeContainer.tsx

```diff
@@ -45,8 +45,8 @@ class HomeContainer extends React.Component<IProps, null> {
           <h1>Alchemy</h1>
           <h2>Budgeting and resource allocation for decentralized organizations</h2>
           <div className={css.topCta}>
-            { process.env.NODE_ENV == 'production'
-                ? <a href='https://alchemy.daostack.io/dao/0xa3f5411cfc9eee0dd108bf0d07433b6dd99037f1'>View Alchemy</a>
+            { (process.env.NODE_ENV === 'production' || process.env.NODE_ENV === 'dev-profiling')
```

Comment:
why do you want the View Alchemy link to point to production on dev-profiling?

Reply:
I don't. See the line below :)

Reply:
even relative though doesnt make sense because the DAO address is fixed and network dependent, so that still only works on production. what we actually want is for every environment except production (but including staging, which is the main change that needs doing) to point to /daos

Reply:
the reasoning is to make de `dev-profiling` behavior as similar to production as possible

[that said, I think it is better to have separate flags/env variables for these settings, i.e. isntead of having this depend on NODE_ENV, we can add a new env variable called MAIN_DAO_ADDRESS or something]

Reply:
ok, I see what you mean now.. Fixed this.